core/camel-api/pom.xml

```diff
@@ -54,6 +54,12 @@
             <artifactId>slf4j-api</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Since JAXB annotations are only needed when using XML legacy parsing, then I wonder if the scope should be optional.

Another goal for the future is to get rid of JAXB annotations all together  - but that is another battle.

Reply:
Do you mean we could set them to use the `provided` scope?

Reply:
Yeah provided would be correct